tests/rules/data/query-plan-analyzer.php

```diff
@@ -72,4 +73,10 @@ public function bug442(Connection $conn, string $table)
         $query = "SELECT email, adaid FROM `$table` WHERE adaid = ?";
         $conn->fetchAssociative($query, [1]);
     }
+
+    public function staticCallTests(): void
+    {
+        StaticDatabase::query("SELECT * FROM `ada` WHERE email = 'test@example.com';");
+        StaticDatabase::executeQuery("SELECT *,adaid FROM `ada` WHERE email = 'test@example.com';");
```

Comment:
to reduce the changed files, you could do a fully-qualified call here

```suggestion
        staabm\PHPStanDba\Tests\Fixture\StaticDatabase::query("SELECT * FROM `ada` WHERE email = 'test@example.com';");
        staabm\PHPStanDba\Tests\Fixture\StaticDatabase::executeQuery("SELECT *,adaid FROM `ada` WHERE email = 'test@example.com';");
```

and remove the namespace import